docs/platform/howto/static-ip-addresses.rst

```diff
@@ -117,8 +117,8 @@ Remove static IP addresses
 
 .. note::
 
-    To dissociate an IP from a service, the service must either have enough IPs
-    still available, or the ``static_ips`` configuration setting for the
+    To dissociate an IP from a service, the service must either have enough IP addresses
+    still available, or the ``static_IP addresses`` configuration setting for the
```

Comment:
Not sure this is correct :D

Reply:
Thank you for catching this one @ftisiot! 😅🙇‍♂️